README.md

```diff
@@ -60,6 +60,30 @@ Protractor flake defaults to using the `standard` parser, which will typically p
 
 You can override this with the `parser` option, specifying one of the [built in parsers](src/parsers/index.js).
 
+#### Cucumberjs parser
```

Comment:
This is a really comprehensive section, but it also hides some of the important parts of the README. How would you feel about moving it to a `cucumber.md` file in new directory called `docs` and linking to it? from the README?

Perhaps something like 

```
#### Parser Documentation

- [cucmber](docs/cucumber.md)